### PR TITLE
Added flags on individual components

### PR DESCRIPTION
**Pull request by yildizozgur**
To be able to manage them during deployments on platforms that do not support hooks.

Note: I am willing to squash the commits after your review.

### CODE REVIEW

File: sentry/templates/configmap-memcached.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
+{{- if .Values.sourcemaps.enabled }}
```
Comment (tewfik-ghariani) [inline sentry/templates/configmap-memcached.yaml:2]: This should be `memcached` right?

Comment (yildizozgur) [reply sentry/templates/configmap-memcached.yaml:2]: `memcache` configured by `sourcemaps` on Chart.yaml. I used the same flag

**Review by tewfik-ghariani — APPROVED** [review]

**Comment by yildizozgur** [comment]
Hi @Mokto , Can you check this improvement? It is crucial for the platform that are not support hooks. This will make the upgrade process easier for the sentry.io chart.

**Review by Mokto — APPROVED** [review]

File: sentry/README.md
Comment (Mokto) [inline sentry/README.md:1]: Actually can you remove the formatting changes in that file please?

Comment (yildizozgur) [reply sentry/README.md:1]: Hi @Mokto,
Thank you very much for your reply. I have removed the formatting in the README.md file.

File: sentry/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sentry
 description: A Helm chart for Kubernetes
 type: application
-version: 22.1.1
+version: 22.2.0
```
Comment (Mokto) [inline sentry/Chart.yaml]: Remove that please

File: sentry/README.md
Comment (Mokto) [inline sentry/README.md:1]: There is formatting in that file too.

Comment (yildizozgur) [reply sentry/README.md:1]: Hi @Mokto, 
Could you explain a little bit more. I just populated from one existing definition. I have checked on preview , I could not see the place I should fix it. 
Do you mean to remove all new entries? Or align them? If yes, can you describe with an example? Thanks.

Comment (Mokto) [reply sentry/README.md:1]: Sorry about that. Everything is fine.

File: sentry/Chart.yaml (hunk 2 above)
Comment (yildizozgur) [reply sentry/Chart.yaml]: Hi @Mokto ,
I have removed this change. Thanks a lot.

File: sentry/README.md
Comment (yildizozgur) [reply sentry/README.md:1]: no problem, thanks a lot

**Review by tewfik-ghariani — APPROVED** [review]

**Review by Mokto — APPROVED** [review]

**Pull request merged**